content/millennium/r4/proprietary-codes.md

```diff
@@ -168,6 +175,18 @@ This code set may include Marital Statuses such as Divorced and Married.
       "userSelected": true
     }
 
+#### Code Set 40 Person Relationship
```

Comment:
person relationships. change anchor above to match

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/56622c4899ac4a5fc33143311dd4cab6c10a7497#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R180

Reply:
need to change "#code-set-40-person-relationship" above to relationships

Reply:
changed to relationships
https://github.com/cerner/fhir.cerner.com/pull/254/commits/a61676ce8e25adc46975ea912b5016edde8a8b6e#diff-f1a7812a3d6e9f4049fb6005eecfb1f5L78